runtime/onert/odc/MinMaxReader.cc

```diff
@@ -25,7 +25,34 @@ namespace
 void inline readMMFile(void *ptr, size_t size, size_t count, FILE *fp, const std::string &err_msg)
 {
   if (fread(ptr, size, count, fp) != count)
+  {
+    std::fclose(fp);
     throw std::runtime_error(err_msg);
+  }
+}
+
+void inline checkHeader(FILE *file)
```

Comment:
(Optional)
You can remove `inline` keyword.
According to https://en.cppreference.com/w/cpp/language/inline, Because the meaning of the keyword inline for functions came to mean "multiple definitions are permitted" rather than "inlining is preferred" since C++98, that meaning was extended to variables.
Therefore, using `inline` keyword within anonymous(unnamed) namespace scope in a cpp file is meaningless.